spec/controllers/requests_controller_spec.rb

```diff
@@ -38,7 +38,7 @@
     describe "GET #index" do
       it "assigns all requests as @requests" do
         get :index, session: valid_session
-        expect(assigns(:requests)).to eq([@a_request])
+        expect(response).to redirect_to(root_url)
```

Comment:
I had already tried to make it work with `valid_session`, but couldn't get it to work (and I couldn't find an example of how to do this with `cancancan`, which handles permissions). I do think it would be nice not to change this test, but it kind of is already covered in the feature spec.
I did forget to rename the test, though.